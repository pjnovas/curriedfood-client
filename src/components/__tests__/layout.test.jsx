import Layout from '../layout';

takeSnapshots(
  {
    default: {
      children: 'Content',
      style: {
        color: 'red'
      }
    }
  },
  Layout
);
