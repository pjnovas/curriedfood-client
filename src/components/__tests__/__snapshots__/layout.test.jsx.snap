// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#263238",
        "flex": 1,
      },
      Object {
        "color": "red",
      },
    ]
  }
>
  Content
</View>
`;
